.github/workflows/integration_tests.yml

```diff
@@ -255,6 +256,59 @@ jobs:
             filelist="targettests/*/*.cpp"
             echo "testlist=$filelist" >> $GITHUB_OUTPUT
           fi
+      
+      - name: Setup anyon account
+        # This step is currently bypassed during nightly runs due to
+        # maintenance. Restore the if check to the original value when
+        # maintenance is complete.
+        #if: github.event_name == 'schedule' || inputs.target == 'nightly' || inputs.target == 'anyon'
+        if: inputs.target == 'anyon'
+        run: |
+          curl -X POST -H "Content Type: application/json" -d '{ "username":"${{ secrets.BACKEND_LOGIN_USERNAME }}","password":"${{ secrets.BACKEND_LOGIN_PASSWORD }}" }' https://api.anyon.cloud/login > credentials.json
```

Comment:
Just noting this may fail until we get an account established and put into our secrets.